benchmarks/user/README.md

```diff
@@ -221,6 +223,61 @@ or to keep the memory footprint under control:
 watch numastat -m
 ```
 
+## Benchmarks' output
+
+The GFlops/s and GPoints/s performance, Operational Intensity (OI) and 
+execution time are emitted to standard output at the end of each run. 
+Further, when running in bench mode, a `.json` file is produced
+(see `python benchmark.py bench --help` for more info) in a folder named
+`results` except if otherwise specified with the `-r` option specifying
+the results directory.
+
+So the isotropic acoustic wave forward Operator in a `512**3` grid, space order
+12, and a simulation time of 100ms:
+
+```
+`DEVITO_LOGGING=DEBUG` python benchmark.py bench -P acoustic -d 512 512 512 -so 12 --tn 100
+```
+
+## Generating a roofline model
+
+To generate a roofline model from the results obtained in `bench` mode,
+one can execute `benchmark.py` in `plot` mode. For example, the command
+
+```
+python benchmark.py plot -P acoustic -d 512 512 512 -so 12 --tn 100 -a aggressive
```

Comment:
this command is currently split over three lines of the .md file; should be a one liner

Reply:
FIXED